src/main/java/monitoringsystemturbo/presenter/ExportPresenter.java

```diff
@@ -88,6 +96,69 @@ public void onConfirm() {
         }
     }
 
+
+    @FXML
+    public void onToNow() {
+        if (toNowCheckBox.isSelected()) {
+            toDatePicker.setValue(now().plusYears(1));
+            toTimePicker.setValue(LocalTime.now());
+            toDatePicker.setEditable(false);

```

Comment:
think about getting some lines together and extracting it to a method. Maybe in this way you will be able to reuse some code